### ISSUE TITLE
Activator Module + Xpetrified Orb: Somewhere for the XP to go?

### ISSUE THREAD

**Issue by TFenby** [desht/ModularRouters#221]
### Describe the feature

A player right clicking with a Forbidden & Arcanus Xpetrified Orb uses (destroys) the orb and gives the player XP. Automating the right click with an Activator Module seems to just lose the XP. That makes sense, since the Router doesn't have an XP bar. I'd love to be able to automate this though. A few ways it might be handled:
- Give the Routers an internal XP buffer
- Give the Routers an internal fluid buffer that can be filled with liquid XP
- Add an option to the Activator to push any resultant items/fluids/XP/etc. to adjacent inventories (similar to the Vacuum)

### Reasons why it should be considered

I can't think of any other items that result in XP/fluids/items being created internal to the player's inventory/XP when you activate them via right click, but if there are, this would also allow those to be automated. 

### Additional details

[Xpetrified Orb](https://wiki.valhelsia.net/external-mods/forbidden-and-arcanus/wiki/items#xpetrified-orb)

**Comment by desht** [comment]
TBH, having the fake player just drop any XP it receives as XP orbs on the ground is likely the best option. The other options are far too effort just to support one single mod (or even a limited set of mods).

**Comment by TFenby** [comment]
Makes sense! I thought option 3 might be doable since the vacuum module kind of already does it. And just anything in general (XP/items/fluid) made by using an item could overflow into adjacent storage.
